install_config/topics/configuring_azure_authorization.adoc

```diff
@@ -0,0 +1,77 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-permissions_{context}']
+= Configuring authorization for Microsoft Azure
+
+*Azure roles*
+
+Configuring Microsoft Azure for {product-title} requires the following role:
+
+[cols='1,3']
+|===
+
+|Contributor
+|To create and manage all types of Microsoft Azure resources.
+
+|===
+
+See the
+link:https://docs.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles[Classic
+subscription administrator roles vs. Azure RBAC roles vs. Azure AD administrator
+roles documentation] for more information.
+
+*Permissions*
+
+Configuring Microsoft Azure for {product-title} requires a service principal,
+which allows the creation and management of Kubernetes service
+load balancers and disks for persistent storage. The service principal values
+are defined at installation time and deployed to the Azure configuration file, located at `/etc/origin/cloudprovider/azure.conf` on {product-title} master and node hosts.
+
+.Procedure
+
+. Using the Azure CLI, obtain the account subscription ID:
++
+[subs=+quotes]
+----
+# az account list
+[
+{
+  "cloudName": "AzureCloud",
+  "id": "<subscription>", <1>
+  "isDefault": false,
+  "name": "Pay-As-You-Go",
+  "state": "Enabled",
+  "tenantId": "<tenant-id>",
+  "user": {
+    "name": "admin@example.com",
+    "type": "user"
+  }
+]
+----
+<1> The subscription ID to use to create the new permissions.
+
+. Create the service principal with the
+role of contributor and with the scope of the Microsoft Azure subscription and
```

Comment:
The contributor role is Azure, yah? 
_Create the service principal with the Microsoft Azure
role of Contributor and scope of Subscription and Resource groups._ 
(is this two scopes or one?

Reply:
1 scope just specifically assigned to the scope of subscription id with a specific resource group. Just being super exact with the privs

Reply:
Clarified.